www/views/setup.html

```diff
@@ -392,7 +392,7 @@ <h1 data-i18n="Notifications">Notifications</h1>
 									<br>
 								</div>
 								<div class="span6">
-									<h2>Pushsafer (iOS/Android/Windows10/Chrome/Firefox):</h2>
+									<h2>Pushsafer (iOS/Android/Windows10) + Image:</h2>
```

Comment:
With this change we need to change all language translations as well
I dont think it is really needed correct?